### PR TITLE
Support Version 2.0.0 changes for regard-application

### LINKED ISSUE #136: Support Version 2.0.0 changes for regard-application
1. Generate server stub for the version 2.0.0 specification 
2. If the api/openapi.yaml is not updated as per the published version 2.0.0 , then copy the api/openapi.yaml file from the generated stub to the source code
3. no change required in the method signature

Following tasks needs to be completed to support regard-application version 2.0.0 , 

- [ ] https://github.com/openBackhaul/OperationKeyManagement/issues/133
- [ ] https://github.com/openBackhaul/OperationKeyManagement/issues/134
- [ ] https://github.com/openBackhaul/OperationKeyManagement/issues/135

### PR DESCRIPTION
**Pull request by vanithavalluripalli9**
Fixes #136

### CODE REVIEW

**Comment by vanithavalluripalli9** [comment]
Reviewer is @DanaSunal

File: server/api/openapi.yaml [hunk 1]
```diff
@@ -7137,7 +7137,6 @@ components:
       type: object
       properties:
         application-name:
-          pattern: "([A-Z][a-z0-9]+){2,}"
```
Comment (DanaSunal) [inline server/api/openapi.yaml:7140]: Is this part of spec already?

**Review by DanaSunal — CHANGES_REQUESTED** [review]

File: server/package.json [hunk 2]
```diff
@@ -23,8 +23,8 @@
     "oas3-tools": "^2.2.3",
     "randexp": "^0.5.3",
     "response-time": "^2.3.2",
-    "onf-core-model-ap-bs": "~1.0.1",
-    "onf-core-model-ap": "~1.0.1"
+    "onf-core-model-ap-bs": "~2.0.0-alpha.3",
+    "onf-core-model-ap": "~2.0.0-alpha.3"
```
[the remark this answers is not in the document]
Comment (vanithavalluripalli9) [reply server/package.json:27]: Done

File: server/api/openapi.yaml (hunk 1 above)
Comment (vanithavalluripalli9) [reply server/api/openapi.yaml:7140]: No , it was added by mistake . So , removed it.

**Review by DanaSunal — APPROVED** [review]

**Pull request merged**